packages/terra-navigation-prompt/CHANGELOG.md

```diff
@@ -3,6 +3,14 @@ Changelog
 
 Unreleased
 ----------
+### Changed
+* Changed 'NavigationPromptCheckpoint' to have the same logic as 'notification-dialog' changes
+  * 'message' prop has been deprecated, will resolve to 'startMessage'
+  * Added a 'startMessage' string prop that will display a message after 'title'
+  * Added a 'content' node prop that will display the given node in the Dialog after 'startMessage'
+  * Added an 'endMessage' string prop that will display a message after 'content'
+  * Added 'buttonOrder' prop to determine order of accept and reject buttons
```

Comment:
These added props should probably be moved into an `### Added` change log section

Reply:
I wasn't sure what the best way to display these changes are for NavigationPrompt since it is more "The exposed API changed" and I was trying to display the API changes as a sublist.

Reply:
Ah, I missed the indentation